conf/modules.config

```diff
@@ -412,4 +412,36 @@ process {
             pattern: '*.flagstat'
         ]
     }
+
+    withName: PRINSEQPLUSPLUS {
+        ext.args =  [
+                params.metagenomics_prinseq_mode == 'dust' ? "-lc_dust=${params.metagenomics_prinseq_dustscore}" : "-lc_entropy=${params.metagenomics_prinseq_entropy}",
+                "-trim_qual_left=0 -trim_qual_left=0 -trim_qual_window=0 -trim_qual_step=0",
+            ].join(' ').trim()
+        ext.prefix = { "${meta.id}_${meta.run_accession}" }
+        publishDir = [
+            [
+                path: { "${params.outdir}/prinseqplusplus/" },
+                mode: params.publish_dir_mode,
+                pattern: '*{_good_out.fastq.gz,_good_out_R1.fastq.gz,_good_out_R2.fastq.gz}',
+                enabled: params.metagenomics_complexity_generatefastq
```

Comment:
Can you check the other parameters in `nextflow.config` in eager we might be using the `save` verb rather than `generate`...